### PR TITLE
66 67 combining

### PR DESCRIPTION
**Pull request by jamestewartjr**
Week of work...

### CODE REVIEW

**Comment by jamestewartjr** [comment]
@tjfwalker

File: models/user.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
     email: DataTypes.STRING,
     email_verified: DataTypes.BOOLEAN,
     password: DataTypes.STRING
   }, {
     classMethods: {
       associate: function(models) {
+        console.log(models)
         // associations can be defined here
+        models.User.hasMany( models.UserTopic )
+        models.User.belongsToMany( models.Topic, { through: 'UserTopics' } )
```
Comment (jrob8577) [inline models/user.js]: I don't think we need the relationship to exist in this direction - we're only concerned with the UserTopics a given user has; the reverse relationship does not make sense.

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -21,3 +21,15 @@
   text-decoration: line-through;
   color: #777;
 }
+
+.button {
```
Comment (jrob8577) [inline public/stylesheets/style.css]: Please use bootstrap styles

File: routes/help.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const express = require( 'express' )
+const router = express.Router()
+
+router.get('/', ( request, response ) => {
+  response.redirect( 'help/howtos' )
+})
+
+router.get('/howtos', ( request, response ) => {
+  const User = request.app.get('models').User
+  console.log('users are here ------------------', User)
```
Comment (jrob8577) [inline routes/help.js]: remove `console.log`

File: routes/help.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+const express = require( 'express' )
+const router = express.Router()
+
+router.get('/', ( request, response ) => {
```
Comment (jrob8577) [inline routes/help.js]: Add space between parens and argument

File: routes/help.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  response.redirect( 'help/howtos' )
+})
+
+router.get('/howtos', ( request, response ) => {
```
Comment (jrob8577) [inline routes/help.js]: Add space between parents and parameter

File: routes/help.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  const User = request.app.get('models').User
+  console.log('users are here ------------------', User)
+  const Topic = request.app.get('models').Topic
+  const UserTopic = request.app.get('models').UserTopic
+
+  const {users} = request.body
```
Comment (jrob8577) [inline routes/help.js]: Space between `users` and braces

File: routes/help.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+  response.render( 'help/howtos', { topics: users } )
```
Comment (jrob8577) [inline routes/help.js]: Confused by the intent here.

File: public/javascripts/help.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+    // const element = $( event.target)
+    // const topic_id = element.data( 'id' )
+    // const viewed = ! element.userTopics( 'viewed')
+
+    $( '.topic-content iframe' ).attr( 'src', $(this).data( 'url' ) )
+    $( '.topic-content .content' ).text( $(this).data( 'content' ) )
+
+
+    //insert association
```
Comment (jrob8577) [inline public/javascripts/help.js]: Remove comments (missing implementation?)

**Comment by jrob8577** [comment]
Closing this in favor of #85

**Pull request closed without merging**